src/components/Backgrounds/Black/Black.scss

```diff
@@ -0,0 +1,4 @@
+.noneContent {
```

Comment:
Remove Black.scss and jut add to the clasName in the Black.tsx file instead of noneContent